src/components/Avatar/Avatar.tsx

```diff
@@ -202,11 +202,9 @@ const Avatar: React.FC<AvatarProps> & {
   );
 };
 
-Object.assign(Avatar, {
+export default withStaticProps(Avatar, {
```

Comment:
this hides some issues:
1. the static properties are still optional.
2. the colors values aren't autocompleted

Reply:
Fixed colors values